### PR TITLE
Remove camera info missing errors when calibration is `None`

### PR DESCRIPTION
**Pull request by snosenzo**
**User-Facing Changes**

- Remove occasional camera info missing errors when calibration is `None`

**Description**
Also consolidated the setting of the image topic so that we can take advantage of it in the subclass of imagemode.

Fixes: FG-5458

### CODE REVIEW

File: packages/studio-base/src/panels/ThreeDeeRender/stories/ImageMode/ImageMode.stories.tsx [hunk 1]
```diff
@@ -696,8 +696,8 @@ export const UnsupportedEncodingError: StoryObj = {
   play: async () => {
     const errorIcon = await waitFor(async () => {
       const icons = await screen.findAllByTestId("ErrorIcon");
-      if (icons.length !== 2) {
-        throw new Error("Expected 2 error icons");
+      if (icons.length !== 1) {
+        throw new Error("Expected l error icon. (unsupported encoding)");
```
Comment (jtbandes) [inline packages/studio-base/src/panels/ThreeDeeRender/stories/ImageMode/ImageMode.stories.tsx:700]:
```suggestion
        throw new Error("Expected 1 error icon. (unsupported encoding)");
```

File: packages/studio-base/src/panels/ThreeDeeRender/renderables/ImageMode/ImageMode.ts [hunk 2]
```diff
@@ -750,7 +755,7 @@ export class ImageMode
    */
   #updateViewAndRenderables(): void {
     const cameraInfo = this.#cameraModel?.info;
-    if (!cameraInfo) {
+    if (!this.#fallbackCameraModelActive() && !cameraInfo) {
```
Comment (jtbandes) [inline packages/studio-base/src/panels/ThreeDeeRender/renderables/ImageMode/ImageMode.ts:758]: So I'm guessing this is the main fix... why did this only happen sometimes and not all the time?

**Review by jtbandes — APPROVED** [review]

**Comment by jtbandes** [comment]
Another question: did this regress at some point? I seem to remember there was a time when these errors weren't showing up...

File: packages/studio-base/src/panels/ThreeDeeRender/renderables/ImageMode/ImageMode.ts (hunk 2 above)
Comment (snosenzo) [reply packages/studio-base/src/panels/ThreeDeeRender/renderables/ImageMode/ImageMode.ts:758]: I think there's a race condition. I'm not able to reproduce it consistently

**Comment by snosenzo** [comment]
> Another question: did this regress at some point? I seem to remember there was a time when these errors weren't showing up...

Also unsure about that, it does have to do with timing and how we decode the images.

**Pull request merged**